content/docs/overview/identity-and-access-management/index.md

```diff
@@ -70,4 +70,41 @@ To set your `Tenant Display Name`:
 
 #### Verified Domain 
 
-The second way to set your tenancy name is to [get your domain verified](../../beyond-the-basics/verified-domain) by the RKVST team. Your verified domain name will be visible to the people you share information with and will be publicly available if you create a Public Asset.
\ No newline at end of file
+The second way to set your tenancy name is to [get your domain verified](../../beyond-the-basics/verified-domain) by the RKVST team. Your verified domain name will be visible to the people you share information with and will be publicly available if you create a Public Asset.
+
+## Enterprise Single Sign-On
+
+[Enterprise customers](https://www.rkvst.com/pricing/) may use their preferred Identity Provider (IDP) to sign-on to RKVST.
```

Comment:
I think its important to state before you can configure an esso, the tenancy needs to have a verified domain?

Also they need to have a cap of >0.

Without both of those the configuration options won't show in the tenancy tab.

I am not sure if that is for docs under an `enable` section, or whether we just point them to pricing and they contact us and we tell them that information

Reply:
I think that the configuration options not showing are covered by the message in the UI, which directs people to RKVST support (for verified domain) and the pricing page (for ESSO). 

I did add a note that they must have a verified domain before setting up SSO, that was a good catch.

Reply:
i agree :) great stuff!